CleanWeather/Scenes/FavouriteCities/FavouriteCitiesWorker.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FavouriteCitiesWorker.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 05/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+protocol FavouriteCitiesWorker {
+    func fetchCity(completion: FetchCitiesCompletion?)
```

Comment:
neeed to start using All or Favourities
because we might be confused what kind of city we fetch